lib/puppet/functions/mysql_password.rb

```diff
@@ -1,21 +0,0 @@
-# @summary
```

Comment:
Removing this was a *bad* idea since https://github.com/puppetlabs/puppetlabs-mysql/blob/0a8f0054113c68b60f19e381f9452155b937b31b/lib/puppet/parser/functions/mysql_password.rb still exists...

Reply:
Fixed in https://github.com/puppetlabs/puppetlabs-mysql/pull/1299